sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -35,6 +35,9 @@
 #define MAX_HDMI_NUM 4
 #define SDW_DMIC_DAI_ID 4
 
+/* 8 combinations with 4 links + unused group 0 */
```

Comment:
@RanderWang I would change the commit message to say something like this: @plbossart please correct me if I said something wrong here. 
"SDW link addr has been updated to include the endpoint descriptor which also holds information about whether the slave should be considered standalone or belonging to an aggregated group. When there are no aggregated slave groups, there is a 1:1 mapping between the number of slaves and the number of BE DAI links. But when slaves are aggregated in groups, the number of BE DAI links equals the number of unique groups plus the number of standalone slaves. Also, when slaves belonging to the same aggregated group are distributed across multiple SDW links, they require multiple CPU DAIs depending on the number of links"

Reply:
perfect! @ranj063

Reply:
@RanderWang Probably best to just use the commit summary text "check change "ASoC: soc-acpi: expand description of _ADR-based" and leave the commit-id out. This patch is not yet merged to upstream, so the commit-id will be meaningless when sent upstream.